Marlin/Configuration.h

```diff
@@ -585,10 +593,25 @@
  * When changing speed and direction, if the difference is less than the
  * value set here, it may happen instantaneously.
  */
-#define DEFAULT_XJERK                 15.0/2 //half jerks
-#define DEFAULT_YJERK                 15.0/2
-#define DEFAULT_ZJERK                  0.4/2
-#define DEFAULT_EJERK                  5.0
+
+#if defined(AON_DEBUG_MODE)
```

Comment:
This could be cleaned up to have less repetition, but it's fine for now I suppose (although we should strive to not have any differences in machine mechanics from testing to production).